mitosheet/src/mito/components/endo/celleditor/cellEditorUtils.tsx

```diff
@@ -125,14 +125,17 @@ export const getStartingFormula = (
             editingMode: editorState.editingMode
         }
     }
+
+    //  Default value is "entire_column" if previouse editingMode is not stored
+    const storedEditingMode= localStorage.getItem("editingMode") === "specific_index_labels" ? "specific_index_labels" : "entire_column"
```

Comment:
This variable gets the editing mode if saved in the local storage. Default Value would be "entire_colum"